vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/LMap.java

```diff
@@ -109,6 +109,14 @@ public void setViewPoint(final LCenter viewpoint)
 			+ viewpoint.getZoom()
 			+ ");");
 	}
+
+	public void centerAndZoom(final LPoint noPoint, final LPoint sePoint)
```

Comment:
Please add a minimal description. Maybe add the link to https://leafletjs.com/reference.html#map-fitbounds.
noPoint and sePoint seems not very descriptive.